api/envoy/config/filter/http/transcoder/v2/transcoder.proto

```diff
@@ -23,7 +23,7 @@ message GrpcJsonTranscoder {
     bytes proto_descriptor_bin = 4;
   }
 
-  // A list of strings that supplies the service names that the
+  // A list of strings that supplies the service names (format: package.service_name ) that the
```

Comment:
Can you also add "fully qualified" before service names? (i.e. "fully qualified service names"). The format LGTM.